lib/krypto/challenge.rb

```diff
@@ -6,7 +6,7 @@
 
 module Krypto
   class Challenge
-    def self.generate(signing_key, challenge_id, challenge_data, request_data)
+    def self.generate(signing_key, challenge_id, challenge_data, request_data, unix_timestamp)
```

Comment:
This changes the call signature, right? The ruby way would be to use something like `timestamp: Time.now` which is an optional kwarg.

(And then lower down `timestamp: timestamp.to_i`)

Reply:
👍 updated